plugins/modules/alert_settings.py

```diff
@@ -0,0 +1,242 @@
+#!/usr/bin/python
+# Copyright: (c) 2024, Dell Technologies
+
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+"""Ansible module for managing alert settings on PowerScale"""
+
+from __future__ import absolute_import, division, print_function
+
+__metaclass__ = type
+
+DOCUMENTATION = r'''
+---
+module: alert_settings
+version_added: '3.2.0'
+short_description: Manage alert settings on a PowerScale Storage System
+description:
+- Managing alert settings on a PowerScale system includes retrieving details of
+  alert settings and enabling or disabling the alert settings.
+
+extends_documentation_fragment:
+  - dellemc.powerscale.powerscale
+
+author:
+  - Bhavneet Sharma (@Bhavneet-Sharma) <ansible.team@dell.com>
+options:
+  enable_celog_maintenance_mode:
+    description:
+    - Enabling CELOG maintenance mode will start a CELOG maintenance window.
+    - During a CELOG maintenance window, the system will continue to log
+      events, but no alerts will be generated.
+    - You will have the opportunity to review all events that took place
+      during the maintenance window when disabling maintenance mode.
+    - Active event groups will automatically resume generating alerts when
+      the scheduled maintenance period ends.
+    type: bool
+  prune:
+    description:
+    - Removes all maintenance mode history that is greater than set number of days.
+    - Range of I(prune) is 0 to 4294967295.
+    type: int
+
+notes:
+- The I(check_mode) and idempotency is supported.
+'''
+
+EXAMPLES = r'''
+- name: Get alert setiings
+  dellemc.powerscale.alert_settings:
+    onefs_host: "{{ onefs_host }}"
+    port_no: "{{ port_no }}"
+    api_user: "{{ api_user }}"
+    api_password: "{{ api_password }}"
+    verify_ssl: "{{ verify_ssl }}"
+
+- name: Enable CELOG maintenance mode
+  dellemc.powerscale.support_assist:
+    onefs_host: "{{ onefs_host }}"
+    port_no: "{{ port_no }}"
+    api_user: "{{ api_user }}"
+    api_password: "{{ api_password }}"
+    verify_ssl: "{{ verify_ssl }}"
+    enable_celog_maintenance_mode: true
+    prune: 1
+
+- name: Disable CELOG maintenance mode
+  dellemc.powerscale.support_assist:
```

Comment:
Please change the file name from support_assist to alert_settings

Reply:
Done.